openpype/hosts/blender/plugins/publish/validate_camera_zero_keyframe.py

```diff
@@ -1,9 +1,11 @@
 from typing import List
 
 import mathutils
+import bpy
 
 import pyblish.api
-import openpype.hosts.blender.api.action
+from openpype.api import ValidateContentsOrder
+from openpype.hosts.blender.api.action import SelectInvalidAction
```

Comment:
I'm not sure this code style change is the best move. Especially since the majority of the OpenPype code base doesn't seem to import these like these for the other hosts, etc.

Reply:
Ok, agree, thanks for the reviews :)